src/mixins/borrow/cooksV2.js

```diff
@@ -60,6 +60,10 @@ export default {
     isApe() {
       return this.chainId === 1 && this.selectedPool.id === 39;
     },
+
+    isSUSDT() {
```

Comment:
- typo
- not open close principle

Reply:
this file still use old config - where it will be fixed